### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by pjol**
[Ticket](https://trello.com/c/zAPqwaOh/27-m-update-schema-to-include-id-field-as-well-as-productid-field)

- Added id field to schema
- Fixed router bug in cases where no items were found

Please review at your convenience @LukeLarson2  @justcord10

### CODE REVIEW

File: db/ETL.js [hunk 1]
```diff
@@ -197,6 +198,7 @@ const stylesConstructor = (data) => {
     filter: { product_id: data[1] },
     update: {
       product_id: data[1],
+      id: data[1],
       $push: {
         features: [],
         styles: {
```
Comment (justcord10) [inline db/ETL.js:204]: updated ETL to follow your new schema looks good

**Review by LukeLarson2 — APPROVED** [review]
Nice use of coercion to ensure its a numerical value. fancy!

File: router.js [hunk 2]
```diff
@@ -46,9 +49,12 @@ router.get('/products/:id', (req, res) => {
...
   })
   .then((data) => {
-    if (!data) res.send(404);
-    data.id = data.product_id;
-    res.send(data);
+    if (!data) {
+      res.sendStatus(404);
+    } else {
+      res.send(data);
+      data.id = data.product_id;
```
Comment (justcord10) [inline router.js]: modify the data after sending?

**Review by justcord10 — APPROVED** [review]
removed the data change after the sending of data in router, updated your if blocks to use elses, and updated your schemas. looks good

**Pull request merged**